benchmark/fio_benchmark_cleanup.yml

```diff
@@ -49,6 +50,7 @@
       shell: "ceph daemon mon.{{ ansible_hostname }} config get mon_allow_pool_delete | grep -q false"
       register: allow_pool_delete
       when: fiobench_pool_name is not defined
+      ignore_errors: true
```

Comment:
are we seeing a failure here that needs to ignore_errors?

Reply:
Yes. The inverse condition evaluates to 1 which is not an error, it just means the value is already set to true

Reply:
ahh good point